release/setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls =
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
+long_description = file: README.md
+author = Elijah Newren
+classifiers =
+    Development Status :: 4 - Beta
+    Operating System :: OS Independent
+    Programming Language :: Python
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+
+[options]
+scripts = ../git-filter-repo
```

Comment:
So, I presume people using PyPI would probably be more likely than average to write a python filtering program that imported filter-repo as a module.  As far as I can tell, you are only distributing the git-filter-repo script here, but importing a python module in a file named with hyphens and without the ".py" extension is quite painful.  What about the git_filter_repo.py symlink?

Reply:
Yes! Good catch -- I didn't notice that `git_filter_repo.py` was a module and not just a script, but looks like you've got a `if __name__` block in it so yeah will add that to what's installed.